cmd/ampwrapper/main.go

```diff
@@ -14,9 +16,15 @@ var (
 )
 
 func init() {
+	homedir, err := homedir.Dir()
+	if err != nil {
+		panic(err)
+	}
+
 	dockerArgs = []string{
 		"run", "-i", "--rm", "--name", "ampcli",
 		"-v", "/var/run/docker.sock:/var/run/docker.sock",
+		"-v", fmt.Sprintf("%s/.config/amp:/root/.config/amp:rw", homedir),
```

Comment:
Since our config command allows you to save settings, you should remove `:rq` from the mount.